functions/index.ts

```diff
@@ -245,3 +246,26 @@ functions.database.ref('gamePortal/gameSpec/reviews/{reviewedGameSpecId}/{review
       return starsSummary;
     });
 });
+
+exports.resizeImage =
+functions.storage.object().onChange.onChange(async (event:any) => {
+  console.log("Resizing the image");
```

Comment:
Move the console.log 3 lines down, and also log the path, bucket, imgType, uid, id, etc :)